api_tests/harness.ts

```diff
@@ -1,10 +1,14 @@
-#!/usr/bin/env node
+#!/usr/bin/env ./api_tests/node_modules/.bin/ts-node --project api_tests/tsconfig.json
```

Comment:
Nice! I didn't know you could do that :D

Reply:
Last chance to remove `harness.js` from the typescript stuff :D.